ultradns/common.go

```diff
@@ -46,7 +46,7 @@ func (r rRSetResource) RRSet() udnssdk.RRSet {
 }
 
 func (r rRSetResource) ID() string {
-	return fmt.Sprintf("%s.%s", r.OwnerName, r.Zone)
+	return fmt.Sprintf("%s:%s", r.OwnerName, r.Zone)
```

Comment:
You absolutely sure about this change? looks like it can have some major side effects. Please test thoroughly.

Reply:
yup we tested it thoroughly and we are sure about the change